k8s/contiv-vpp/values.yaml

```diff
@@ -31,6 +32,14 @@ contiv:
     nodeInterconnectCIDR: 192.168.16.0/24
     vxlanCIDR: 192.168.30.0/24
     nodeInterconnectDHCP: false
+    srv6ServicePolicyBSIDSubnetCIDR: 5555::/16
```

Comment:
now as I look at this, it really seems to be worthwhile embedding all SRv6-related settings into a nested level (struct in go) called srv6.

Reply:
fixed in https://github.com/contiv/vpp/pull/1523/commits/0fca64b3f38d65d77e97a43f47e1588a834272f7